spaceship/lib/spaceship/du/utilities.rb

```diff
@@ -16,6 +17,20 @@ def content_type(path)
       return 'video/quicktime' if path.end_with?('.mov')
       return 'video/mp4' if path.end_with?('.m4v')
       return 'video/mp4' if path.end_with?('.mp4')
+      return 'text/plain' if path.end_with?('.txt')
+      return 'application/pdf' if path.end_with?('.pdf')
+      return 'application/msword' if path.end_with?('.doc')
+      return 'application/vnd.openxmlformats-officedocument.wordprocessingml.document' if path.end_with?('.docx')
+      return 'application/rtf' if path.end_with?('.rtf')
+      return 'application/x-iwork-pages-sffpages' if path.end_with?('.pages')
+      return 'application/vnd.ms-excel' if path.end_with?('.xls')
+      return 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet' if path.end_with?('.xlsx')
+      return 'application/x-iwork-numbers-sffnumbers' if path.end_with?('.numbers')
+      return 'application/x-rar-compressed' if path.end_with?('.rar')
+      return 'application/xml' if path.end_with?('.plist')
+      return 'text/x-apport' if path.end_with?('.crash')
+      return 'video/x-msvideo' if path.end_with?('.avi')
+      return 'application/zip' if path.end_with?('.zip')
```

Comment:
We already implicitly depend on the `mime-types` gem. We should declare the dependency explicitly and use that instead of re-implementing it here. If that's not something we want to do, then lets just use a Hash mapping the extensions to the mime types.

Reply:
i have replaced with hash. Since this is a module i have to keep this with the function. Otherwise unwanted global memory.